_includes/home.html

```diff
@@ -1,4 +1 @@
-{% assign home = "" %}
```

Comment:
Removing this will break some use cases. We need to be able to:

Host the site on istio.io
Host the site locally when someone runs Jekyll
Host the site on people's own repos (geeknoid.istio.io)

Making this change will break the third case I believe.